services/ymax-planner/src/pending-tx-manager.ts

```diff
@@ -253,9 +259,93 @@ const gmpMonitor: PendingTxMonitor<GmpTx, EvmContext> = {
   },
 };
 
+const makeAccountMonitor: PendingTxMonitor<MakeAccountTx, EvmContext> = {
+  watch: async (ctx, tx, log, opts) => {
+    await null;
+
+    const { txId, expectedAddr, destinationAddress } = tx;
+    const logPrefix = `[${txId}]`;
+
+    expectedAddr || Fail`${logPrefix} Missing expectedAddr`;
+    destinationAddress ||
+      Fail`${logPrefix} Missing destinationAddress (factory)`;
+
+    const {
+      namespace,
+      reference,
+      accountAddress: factoryAddr,
+    } = parseAccountId(destinationAddress);
+    const caipId: CaipChainId = `${namespace}:${reference}`;
+
+    const provider =
+      ctx.evmProviders[caipId] ||
+      Fail`${logPrefix} No EVM provider for chain: ${caipId}`;
+
+    const watchArgs = {
+      factoryAddr: factoryAddr as `0x${string}`,
+      provider,
+      expectedAddr: expectedAddr as `0x${string}`,
+      log: (msg, ...args) => log(logPrefix, msg, ...args),
+    };
+
+    let walletCreated: boolean | undefined;
+
+    if (opts.mode === 'live') {
+      walletCreated = await watchSmartWalletTx({
+        ...watchArgs,
+        timeoutMs: opts.timeoutMs,
+      });
+    } else {
+      const abortController = new AbortController();
+      const liveResultP = watchSmartWalletTx({
+        ...watchArgs,
+        timeoutMs: opts.timeoutMs,
+        signal: abortController.signal,
+      });
+      void liveResultP.then(found => {
+        if (found) {
+          log(`${logPrefix} Live mode completed`);
+          abortController.abort();
+        }
+      });
+
+      await null;
+
+      const currentBlock = await provider.getBlockNumber();
+      await waitForBlock(provider, currentBlock + 1);
+
+      walletCreated = await lookBackSmartWalletTx({
+        ...watchArgs,
+        publishTimeMs: opts.publishTimeMs,
+        chainId: caipId,
+        signal: abortController.signal,
+      });
```

Comment:
@rabi-siddique [`lookBackSmartWalletTx`](https://github.com/Agoric/agoric-sdk/blob/ec95db1b08afc24f29c8bb2dca92b77cfefe4ea7/services/ymax-planner/src/watchers/wallet-watcher.ts#L128) and [`watchSmartWalletTx`](https://github.com/Agoric/agoric-sdk/blob/ec95db1b08afc24f29c8bb2dca92b77cfefe4ea7/services/ymax-planner/src/watchers/wallet-watcher.ts#L55) both require their argument to be a [SmartWalletWatch](https://github.com/Agoric/agoric-sdk/blob/ec95db1b08afc24f29c8bb2dca92b77cfefe4ea7/services/ymax-planner/src/watchers/wallet-watcher.ts#L46), which has required fields `kvStore` and `txId` that are missing from this `watchArgs`, causing three tsc errors in the above code but also behavioral issues AFAICT.